Sync with official fan translations

Vankog opened the pull request:
status: done
implementation of #2

* Synchronizing the phrases with 
   * the officially listed fan translation PDFs from the CAH website by "Balz" and "Raphael" 
   * and the official US PDF.
* Adding comments: 
   * with contexts & known references 
   * alternative, earlier and original (i.e. by the fan translations above) translations. 
   * the official English phrases
   * => Reasoning: 
      * -> easier search-ability for phrases and words for future check ups
      * -> should prevent duplicates and wrong translations
      * -> matching the official English phrases with one or more translations
* Improving translations
* Moving some phrases to other topics
* Improving line breaks
* Incorporates the card-changing-commit b7e627186d5f29c3085e9da4498600c4d706e97d by @AnotherStranger

Commit:
Neue Basiskarten erstellt.

Vankog committed:
* Synchronizing the white cards with the officially listed fan translations from the CAH website by "Balz" and "Raphael".
* Adding comments with contexts, known references and alternative/earlier/official translations.
   * -> easier searchability for phrases and words
   * -> should prevent duplicates and wrong translations.
* Improving some phrases
* Moving some phrases to other topics

Vankog committed:
* Synchronizing the black cards with the officially listed fan translations from the CAH website by "Balz" and "Raphael".
* Adding comments with contexts, known references and alternative/earlier/official translations.
   * -> easier searchability for phrases and words
   * -> should prevent duplicates and wrong translations.
* Improving some phrases
* Moving some phrases to other topics

Vankog committed:
adding original English phrases for reference

Vankog committed:
comment formatting for better readability

Vankog commented on cah-fullformat-german.tex:
```diff
@@ -43,10 +43,10 @@
 \include{categories/fsk16}
 \include{categories/fsk18}
-\include{categories/politik}
+\include{categories/geschichte}
```
just sorted alphabetically

Vankog commented on commands.tex:
```diff
@@ -138,7 +138,7 @@
 		}%
 	%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-	% White Cards %
+	% Black Cards %
```
fixing wrong comment

Vankog committed:
updating contributors and categories of README.md

Vankog committed:
minor typos and clarifications

Prior99 commented:
Hi, what's the license of the official fan translations? Can we use them without mentioning?

Vankog committed:
"Martin Luther King" -> "Martin Luther", because Martin Luther King is a duplicate -> see "Der Pfarrer Dr. Martin Luther King, Jr."

Vankog commented:
1. It is the same license as the main game. Mentioning must be done.
1. I mentioned them in the README.md for this reason
1. That's one reason why I opened #5 as well

Vankog committed:
change word order of "weisen schwarzen alten Mann." -> "weisen alten schwarzen"

Prior99 commented on categories/basis.tex:
```diff
@@ -21,276 +21,709 @@
-\whitecard{Basis}{Anfangen, zu singen und zu tanzen.}%
+% engl. "Alcoholism"
+\whitecard{Basis}{Alten Leuten zuzwinkern.}%
+% before: zuwinken
```
Remove the lines with `% before`. The new translations are much better and the git history should be enough if anybody cares about the old ones.

Prior99 commented on categories/basis.tex:
```diff
@@ -21,276 +21,709 @@
+\whitecard{Basis}{Das Gedränge.}%
+% alternative to engl. "The Hustle"
+\whitecard{Basis}{Das Glasdach.}%
+% original: Glasdecke
```
Remove the lines with `%original`. See comment above.

Prior99 reviewed (changes requested):
Hey. Wow thanks for your great work, I can see that you put much effort into this. Please remove the lines as suggested by my comments and I'd be more than happy to merge this!

Vankog commented:
I can see why you want to remove the "before" comments. I'll remove them.
However, I'd like to stand up for keeping the "original" ones. These are the ones from the fan translations by Ballz and Raphael. I introduced them to have a mapping. Otherwise, some of their phrases that I changed for one reason or another, might be reintroduced in the future. I'd like to prevent this from happening.
Especially, because they also added some German original phrases that don't show up in the engl. version.

Prior99 commented:
Sounds good!

On Fri, Apr 13, 2018, 12:59 Daniel Mönch <notifications@github.com> wrote:

> I can see why you want to remove the "before" comments. I'll remove them.
> However, I'd like to stand up for keeping the "original" ones. These are
> the ones from the fan translations by Ballz and Raphael. I introduced them
> to have a mapping. Otherwise, some of their phrases that I changed for one
> reason or another, might be reintroduced in the future. I'd like to prevent
> this from happening.
> Especially, because they also added some German original phrases that
> don't show up in the engl. version.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/Prior99/cah-fullformat-german/pull/6#issuecomment-381101348>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABbCdu-l0LCEDioWgLK4gBUV4szOb7j1ks5toIUmgaJpZM4TQh69>
> .
>

Vankog committed:
removing "before" tags Prior99/cah-fullformat-german#6

Vankog commented:
better?

Prior99 commented:
Yes. Accepted. Thanks for your contribution :)
